src/atoms/breadcrumbs/Breadcrumbs.scss

```diff
@@ -1,35 +1,50 @@
 @import '../../../assets/styles/globals';
 
-$breadcrumbs__font-size        : $font-size-small !default;
-$breadcrumbs__line-height      : 1.5 !default;
-$breadcrumbs__item-color       : $blue !default;
-$breadcrumbs__link-color       : $gray-darker !default;
-$breadcrumbs__link-color--hover: $blue !default;
-$breadcrumbs__separator-margin : 0 $spacer !default;
+$breadcrumbs__list-margin                 : 0 !default;
+$breadcrumbs__list-padding                : 0 !default;
+$breadcrumbs__list-item-margin            : 0 0 $spacer 0 !default;
+$breadcrumbs__font-size                   : $font-size-small !default;
```

Comment:
move to top, keep the same ore as it's used in code

Additionally you can add an empty line between a group of vars, it's more readable
```
$breadcrumbs__font-size                   : $font-size-small !default;

$breadcrumbs__list-margin                 : 0 !default;
$breadcrumbs__list-padding                : 0 !default;

$breadcrumbs__list-item-margin            : 0 0 $spacer 0 !default;

$breadcrumbs__link-color                  : $black !default;
$breadcrumbs__link-color--hover           : $blue !default;
$breadcrumbs__link-color--current         : $blue !default;
$breadcrumbs__link-pointer-events--current: none !default;
$breadcrumbs__link-text-decoration        : none !default;

$breadcrumbs__separator-margin            : 0 $spacer !default;
$breadcrumbs__separator-color             : $breadcrumbs__link-color !default;
```